coordinator/src/lib.rs

```diff
@@ -106,3 +107,10 @@ pub fn decimal_from_f32(float: f32) -> Decimal {
 pub fn f32_from_decimal(decimal: Decimal) -> f32 {
     decimal.to_f32().expect("Decimal to fit into f32")
 }
+
+#[derive(Clone, Copy, Debug)]
+pub struct ChannelOpeningParams {
+    pub trader_reserve: Amount,
+    pub coordinator_reserve: Amount,
+    pub external_funding: Option<Amount>,
```

Comment:
hmm.. I kind of like the option better because it is a bit more expressive than automatically implying that a zero amount is a `OpenSingleFundedChannel`

```
match params.external_funding {
    Some(external_funding) => {
        TradeAction::OpenSingleFundedChannel { external_funding }
    }
    None => TradeAction::OpenDlcChannel,
}
```

Reply:
But what does it mean to have a 0-valued `external_funding` then?

Reply:
Probably a bug. Any value below 250k is probably a bug.